### PR TITLE
Rewrite of Google Santa to match current docs

### PR DESCRIPTION
**Pull request by davidbpirie**
Rewrite of manifest for com.google.santa to match the current documentation at https://santa.dev/deployment/configuration.html. Some keys have been re-named, some definitions modified, and many added.

### CODE REVIEW

File: Manifests/ManagedPreferencesApplications/com.google.santa.plist [hunk 1]
```diff
@@ -1,575 +1,995 @@
...
-			<integer>1</integer>
-			<key>pfm_description</key>
-			<string></string>
-			<key>pfm_description_reference</key>
-			<string>1 = MONITOR, 2 = LOCKDOWN, defaults to MONITOR</string>
-			<key>pfm_name</key>
-			<string>ClientMode</string>
-			<key>pfm_range_list</key>
-			<array>
+	<dict>
```
Comment (apizz) [inline Manifests/ManagedPreferencesApplications/com.google.santa.plist:4]: In plists by default the initial `<plist>` & `<dict>` tags are at the same level.  You should remove this unnecessary extra indentation here and in the rest of the subsequent file.

**Review by apizz — CHANGES_REQUESTED** [review]

**Comment by davidbpirie** [comment]
Apologies - that was an auto-format error. Indentation corrected.

**Comment by apizz** [comment]
No worries, thanks for fixing that.

The last 2 things that needs to be addressed here:

1) the `pfm_description_reference` was meant to be the more documentation-replicated description.  However, this _does not_ appear in the GUI.  So any preference should have something in the `pfm_description` at the very least that says what it does / how to generally configure it.

2) is the ordering of preferences in each dictionary.  Some manifests are a bit older and didn't adhere an alphabetical ordering, but I'd love to get this corrected such that the keys in each preference dict are listed A>Z.  So the current `ClientMode` preference dictionary:

```
<dict>
    <key>pfm_title</key>
    <string>Client Mode</string>
    <key>pfm_name</key>
    <string>ClientMode</string>
    <key>pfm_type</key>
    <string>integer</string>
    <key>pfm_description</key>
    <string></string>
    <key>pfm_description_reference</key>
    <string>1 = MONITOR, 2 = LOCKDOWN, defaults to MONITOR</string>
    <key>pfm_default</key>
    <integer>1</integer>
    <key>pfm_range_list</key>
    <array>
        <integer>1</integer>
        <integer>2</integer>
    </array>
    <key>pfm_range_list_titles</key>
    <array>
        <string>Monitor</string>
        <string>Lockdown</string>
    </array>
</dict>
```

Should be:
```
<dict>
    <key>pfm_default</key>
    <integer>1</integer>
    <key>pfm_description</key>
    <string></string>
    <key>pfm_description_reference</key>
    <string>1 = MONITOR, 2 = LOCKDOWN, defaults to MONITOR</string>
    <key>pfm_name</key>
    <string>ClientMode</string>
    <key>pfm_range_list</key>
    <array>
        <integer>1</integer>
        <integer>2</integer>
    </array>
    <key>pfm_range_list_titles</key>
    <array>
        <string>Monitor</string>
        <string>Lockdown</string>
    </array>
    <key>pfm_title</key>
    <string>Client Mode</string>
    <key>pfm_type</key>
    <string>integer</string>
</dict>
```

**Comment by davidbpirie** [comment]
For the most part, I had already populated pfm_documentation, unless the pfm_title was self explanatory. I have had another look through and made a few tweaks where I could see a potential improvement.
I have also re-sorted the keys alphabetically.

**Review by apizz — APPROVED** [review]
Think things are in a good place now, thank you!

**Pull request merged**